client/src/main.rs

```diff
@@ -99,15 +104,35 @@ async fn main() {
 			println!("EtVerifierWrapper contract deployed. Address: {}", w_addr);
 		},
 		Mode::Attest => {
-			let client = EigenTrustClient::new(config, user_secrets_raw);
+			let client = EigenTrustClient::new(config.clone(), user_secrets_raw);
+			println!("Attestations:\n{:#?}", config.ops);
 			client.attest().await.unwrap();
 		},
 		Mode::Verify => {
-			let url = format!("{}/score", config.server_url);
-			let proof_raw: ProofRaw = reqwest::get(url).await.unwrap().json().await.unwrap();
+			println!("Verifying proofs...");
+			let mut proof: Proof = Proof::from(ProofRaw { pub_ins: vec![[0; 32]], proof: vec![0] });
+			let attestations = get_attestations(&config).await.unwrap();
+
+			match mng_store.lock() {
+				Ok(mut manager) => {
+					manager.generate_initial_attestations();
+
+					for att in attestations {
+						manager.add_attestation(att).unwrap();
+						println!("Added attestation");
+					}
+
+					proof = manager.calculate_proofs().unwrap();
+				},
+				Err(e) => {
+					println!("error: {:?}", e);
+				},
+			}
+
 			let client = EigenTrustClient::new(config, user_secrets_raw);
-			client.verify(proof_raw).await.unwrap();
-			println!("Successful verification!");
+
+			client.verify(ProofRaw::from(proof)).await.unwrap();
+			println!("Proof verified");
 		},
```

Comment:
Maybe we should split this into two commands:
- Generate proof and save to file
- Read proof from the file and verify it

We should go in this direction because we will have proofs for multiple different sets that we want to aggregate in different ways

Reply:
Also, we could get rid of some nesting

Reply:
Sure! I kept this in the same action mainly for quick prototyping.

I will implement this to save the latest proof in a file, and that the verify command reads from it and verifies it. However, I think we should discuss wether in the future we save multiple proofs in one file or separate files, and how do we specify which proof to verify.

Reply:
This is yet to be discussed, but roughly every set will have its unique identifier which is going to be used as a file name as well along with a prefix that specifies if its a single set proof or multiple sets aggregated.

For now we can treat the whole network as a single proof and name the file `et_proof.json` or similar.